webpack.config.coffee

```diff
@@ -20,7 +20,7 @@ else
 module.exports =
   cache: true
 
-  devtool: 'source-map' # Always include source maps!
+  devtool: '#inline-source-map' # Always include source maps!
 
```

Comment:
Yep, agreed on not including them inline (because of file size). From the webpack PR it looks like Chrome Canary might have fixed it? https://github.com/webpack/webpack/issues/2145#issuecomment-224368361 or chunking the files into smaller pieces ; (

I think it's important to try to keep the source maps in production though